Producer.py

```diff
@@ -0,0 +1,54 @@
+from kafka import KafkaProducer
+from json import dumps
+from time import sleep
+from pymongo import MongoClient
+from json import loads
+import random
+from flask import Flask, flash, render_template, request, jsonify
+from flask_cors import CORS, cross_origin
+from flask_pymongo import PyMongo
+import sys
+from werkzeug.utils import cached_property
+import logging
+from Event import Event
+from _datetime import datetime
+
```

Comment:
remove line 7 & 8

Reply:
I am using logging, Event class and datetime.